modules/Microsoft.ServiceBus/namespaces/deploy.bicep

```
@@ -200,6 +203,18 @@ module serviceBusNamespace_virtualNetworkRules 'virtualNetworkRules/deploy.bicep
   }
 }]
 
+module serviceBusNamespace_networkRuleSet 'networkRuleSets/deploy.bicep' = if (skuName == 'Premium' || !empty(networkRuleSets)) {
```

Still an open discussion, but so far we don't want to transparently fix wrong configurations. If a user sets the deployment wrong (e.g. sets networkRuleSets on a sku different than Premium) we want the deployment to fail.
```suggestion
module serviceBusNamespace_networkRuleSet 'networkRuleSets/deploy.bicep' = if (!empty(networkRuleSets)) {
```